assets/xml/objects/object_link_child.xml

```diff
@@ -7,63 +7,63 @@
+        <DList Name="gLinkChildRightThighNearDL" Offset="0x204F0"/>
+        <DList Name="gLinkChildRightShinNearDL" Offset="0x206E8"/>
+        <DList Name="gLinkChildRightFootNearDL" Offset="0x20978"/>
+        <DList Name="gLinkChildLeftThighNearDL" Offset="0x20AD8"/>
+        <DList Name="gLinkChildLeftShinNearDL" Offset="0x20CD0"/>
+        <DList Name="gLinkChildLeftFootNearDL" Offset="0x20F60"/>
         <DList Name="gLinkChildCollarNearDL" Offset="0x210C0"/>
         <DList Name="gLinkChildTorsoNearDL" Offset="0x21130"/>
         <DList Name="gLinkChildHeadNearDL" Offset="0x21360"/>
         <DList Name="gLinkChildHatNearDL" Offset="0x219B0"/>
-        <DList Name="gLinkChildLeftShoulderNearDL" Offset="0x21AE8"/>
-        <DList Name="gLinkChildLeftForearmNearDL" Offset="0x21CB8"/>
-        <DList Name="gLinkChildRightShoulderNearDL" Offset="0x21E18"/>
-        <DList Name="gLinkChildRightForearmNearDL" Offset="0x21FE8"/>
+        <DList Name="gLinkChildRightShoulderNearDL" Offset="0x21AE8"/>
+        <DList Name="gLinkChildRightForearmNearDL" Offset="0x21CB8"/>
+        <DList Name="gLinkChildLeftShoulderNearDL" Offset="0x21E18"/>
+        <DList Name="gLinkChildLeftForearmNearDL" Offset="0x21FE8"/>
         
         <!--Limbs-->
-        <Limb Name="gLinkChildLimb_02CDC8" LimbType="LOD" Offset="0x2CDC8"/>
-        <Limb Name="gLinkChildWaistLimb" LimbType="LOD" Offset="0x2CDD8"/>
-        <Limb Name="gLinkChildLimb_02CDE8" LimbType="LOD" Offset="0x2CDE8"/>
-        <Limb Name="gLinkChildLeftThighLimb" LimbType="LOD" Offset="0x2CDF8"/>
-        <Limb Name="gLinkChildLeftShinLimb" LimbType="LOD" Offset="0x2CE08"/>
-        <Limb Name="gLinkChildLeftFootLimb" LimbType="LOD" Offset="0x2CE18"/>
-        <Limb Name="gLinkChildRightThighLimb" LimbType="LOD" Offset="0x2CE28"/>
-        <Limb Name="gLinkChildRightShinLimb" LimbType="LOD" Offset="0x2CE38"/>
-        <Limb Name="gLinkChildRightFootLimb" LimbType="LOD" Offset="0x2CE48"/>
-        <Limb Name="gLinkChildLimb_02CE58" LimbType="LOD" Offset="0x2CE58"/>
-        <Limb Name="gLinkChildHandLimb" LimbType="LOD" Offset="0x2CE68"/>
-        <Limb Name="gLinkChildHatLimb" LimbType="LOD" Offset="0x2CE78"/>
-        <Limb Name="gLinkChildCollarLimb" LimbType="LOD" Offset="0x2CE88"/>
-        <Limb Name="gLinkChildRightSholderLimb" LimbType="LOD" Offset="0x2CE98"/>
-        <Limb Name="gLinkChildRightForearmLimb" LimbType="LOD" Offset="0x2CEA8"/>
-        <Limb Name="gLinkChildLeftHandLimb" LimbType="LOD" Offset="0x2CEB8"/>
-        <Limb Name="gLinkChildLeftSholderLimb" LimbType="LOD" Offset="0x2CEC8"/>
-        <Limb Name="gLinkChildLeftForearmLimb" LimbType="LOD" Offset="0x2CED8"/>
-        <Limb Name="gLinkChildRightHandLimb" LimbType="LOD" Offset="0x2CEE8"/>
-        <Limb Name="gLinkChildSwordAndSheathLimb" LimbType="LOD" Offset="0x2CEF8"/>
-        <Limb Name="gLinkChildTorsoLimb" LimbType="LOD" Offset="0x2CF08"/>
+        <Limb Name="gLinkChildRootLimb" Type="LOD" Offset="0x2CDC8"/>
```

Comment:
```suggestion
        <Limb Name="gLinkChildRootLimb" LimbType="LOD" Offset="0x2CDC8"/>
```
another arbitrary #1044 decision
(you may want to be more careful when merging changes)